Updating intro to metrics query for calendar aligned queries (#21319)
DataDog/documentation: content/en/dashboards/functions/rollup.md

You can customize how your metrics data is bucketed over time when using the `.rollup()` function with calendar aligned queries. This feature allows you the flexibility to define:

> * Calendar aligned monthly queries with adjustable start date / timezones
> _e.g. How did my monthly client errors compare between February and December last year?_

Reviewer: ```suggestion
* Calendar aligned monthly queries with adjustable start date and timezones. For example, you can compare your monthly client errors for February and December of last year.
```

> * Weekly rollups with adjustable start date / timezones
> _e.g. How many weekly transactions are open (if my week starts on Mondays)_

Reviewer: ```suggestion
* Weekly rollups with adjustable start date and timezones. For example, see how many weekly transactions are open (if your week starts on Mondays).
```

> * Daily rollups with adjustable start time / timezones
> _e.g. How many events of interested occurred on the current day? (given my day begins at midnight Pacific Time)_

Reviewer: ```suggestion
* Daily rollups with adjustable start time and timezones. For example, see how many events of interest occurred on the current day (given the day begins at midnight Pacific Time).
```
